pom.xml

```diff
@@ -47,7 +47,7 @@
     </site>
   </distributionManagement>
   <properties>
-    <ddf.version>2.11.2</ddf.version>
+    <ddf.version>2.11.3-SNAPSHOT</ddf.version>
```

Comment:
Do we want to be depending on a SNAPSHOT version of DDF on the release branch?

Reply:
If we want to get the changes to DDF that make this work

Reply:
I think we've been on the snapshot version more than a released version throughout Alliance's life

Reply:
This PR bumps it anyway : https://github.com/codice/alliance/pull/466